lib.Makefile

```diff
@@ -876,6 +876,12 @@ push-image-arch-to-registry-%:
 		$(NOECHO) $(NOOP)\
 	)
 
+# push multi-arch manifest where supported.
+push-manifests: var-require-all-IMAGETAG  $(addprefix sub-manifest-,$(call escapefs,$(PUSH_MANIFEST_IMAGES)))
+sub-manifest-%:
+	$(DOCKER) manifest create $(call unescapefs,$*):$(IMAGETAG) $(addprefix --amend ,$(addprefix $(call unescapefs,$*):$(IMAGETAG)-,$(VALIDARCHES)))
```

Comment:
any specific reason not to restore `MANIFEST_TOOL` and `MANIFEST_TOOL_CMD` exactly as it is in https://github.com/projectcalico/calico/pull/8103/files#diff-abfd44a7d28b43da680f905985a06fe893b7e4aa8920d87fee8070b261645b55L915?

In particular, I believe the `-v $(DOCKER_CONFIG):/root/.docker/config.json` arg will be necessary for credentials when running on semaphore... I had to do something similar for windows manifests: https://github.com/projectcalico/calico/blob/004a6e3bbf0006bab195d1dad217b51d1a948911/lib.Makefile#L1319-L1345

Reply:
The manifest tool is removed from go-build v0.90. docker manifest command (even though still experimental) was [merged into docker cli back in 2017](https://github.com/docker/cli/pull/138). After that, projects from kubernetes, aws, ibm switched to that. I think it would make more sense for us to use the official utility as well.

Reply:
For semaphore build, we do docker login first. docker manifest utility will use the same credential once we login to a registry.

Reply:
Oh I see! Yes, just noticed that the manifest cmd for the windows images is just `docker manifest`